lib/types.js

```diff
@@ -3,14 +3,6 @@
 const { parse } = require('graphql');
 const debug = require('debug')('graphql-component:types');
 
-const iterateObjectTypes = function *(definitions) {
```

Comment:
as much as I like the use of the generator, I couldn't get it to play nice with deleting things from the iterable it was operating on.